modules/gltools/src/state-tracker/unified-parameter-api.js

```diff
@@ -9,12 +9,14 @@ import {
 } from './webgl-parameter-tables';
 
 import {pushContextState, popContextState} from './track-context-state';
-import {isObjectEmpty} from '../utils';
+import {isObjectEmpty, isWebGL, assert} from '../utils';
 
 // Sets any GL parameter regardless of function (gl.blendMode, ...)
 // Note: requires a `cache` object to be set on the context (gl.state.cache)
 // This object is used to fill in any missing values for composite setter functions
 export function setParameters(gl, values) {
+  assert(isWebGL(gl), 'setParameters requires a WebGL context');
```

Comment:
Nit: I think there is an `assertWebGLContext` function (unless you already removed it). 

If there still is, you might want to use either approach consistently across the frameworks.

Reply:
So I pulled `isWebGL` and `isWebGL2` into gltools and left the assert versions in webgl. My reasoning was just to pull the minimal that was necessary to get the context creation working in gltools, but it might be worth pulling the asserts to the lower level too...

Reply:
OK. If it helps make a decision either way, I think the tradeoffs were:
- one hand it is nice to just go with assert. That would set it up for a future babel-plugin that strips all asserts for a smaller production build.
- On the other hand, there is the issue of the error message string being duplicated in tons of files, I think that is why those functions were created instead of using assert.